<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DGT-0F1-LAP070" ignored="4" name="Test" tests="1" failures="1" timestamp="31 Aug 2019 07:19:05 GMT" time="69.237" errors="0">
  <testcase name="login" time="69.177" classname="com.Laundry.PageObjects.Login_TC1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.Laundry.PageObjects.Login_TC1.login(Login_TC1.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login05" time="0.0" classname="com.Laundry.PageObjects.Login_TC3">
    <ignored/>
  </testcase> <!-- login05 -->
  <testcase name="login05" time="0.0" classname="com.Laundry.PageObjects.Login_TC5">
    <ignored/>
  </testcase> <!-- login05 -->
  <testcase name="login02" time="0.0" classname="com.Laundry.PageObjects.Login_TC2">
    <ignored/>
  </testcase> <!-- login02 -->
  <testcase name="login04" time="0.0" classname="com.Laundry.PageObjects.Login_TC4">
    <ignored/>
  </testcase> <!-- login04 -->
</testsuite> <!-- Test -->
